review.go

```diff
@@ -32,17 +52,40 @@ type ReviewListParams struct {
 	CreatedRange *RangeQueryParams `form:"created"`
 }
 
+// ReviewIPAddressLocation represents information about the IP associated with a review.
+type ReviewIPAddressLocation struct {
+	City      string  `json:"city"`
+	Country   string  `json:"country"`
+	Latitude  float64 `json:"latitude"`
+	Longitude float64 `json:"longitude"`
+	Region    string  `json:"region"`
+}
+
+// ReviewSession represents information about the browser session associated with a review.
+type ReviewSession struct {
+	Browser  string `json:"browser"`
+	Device   string `json:"device"`
+	Platform string `json:"platform"`
+	Version  string `json:"session"`
```

Comment:
```suggestion
	Version  string `json:"version"`
```

Reply:
Would you believe me if I said I do this to make sure you review properly? 😓

Reply:
ಠ_ಠ